cleanlab/classification.py

```diff
@@ -398,7 +455,7 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
             # This is less accurate, but best we can do if no sample_weight.
             self.clf.fit(x_cleaned, labels_cleaned, **self.clf_final_kwargs)
 
-        return self.clf
```

Comment:
Are you sure we shouldn't return `self` instead of `self.clf`?

Cf:  https://github.com/scikit-learn/scikit-learn/blob/baf828ca126bcb2c0ad813226963621cafe38adb/sklearn/ensemble/_forest.py#L493

and your LogReg link also returns `self`

Reply:
great catch